<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game_setup</class>
 <widget class="QMainWindow" name="game_setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game setup</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>180</width>
      <height>85</height>
     </rect>
    </property>
    <property name="title">
     <string>Please select game type</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButton">
       <property name="text">
        <string>1 Player (vs computer)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radioButton_2">
       <property name="text">
        <string>2 Player (human vs human)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>395</width>
      <height>173</height>
     </rect>
    </property>
    <property name="title">
     <string>Player information</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>29</y>
       <width>271</width>
       <height>63</height>
      </rect>
     </property>
     <property name="title">
      <string>Player 1</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>29</y>
        <width>117</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Please enter your name </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>108</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>98</y>
       <width>271</width>
       <height>63</height>
      </rect>
     </property>
     <property name="title">
      <string>Player 2</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>29</y>
        <width>114</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Please enter your name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>108</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>307</y>
      <width>395</width>
      <height>112</height>
     </rect>
    </property>
    <property name="title">
     <string>Coin flip</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>29</y>
       <width>381</width>
       <height>44</height>
      </rect>
     </property>
     <property name="text">
      <string>The computer will now randomly select which player will go first (like a coin flip)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>79</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose player</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>413</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
